mne/decoding/receptive_field.py

```diff
@@ -240,14 +271,13 @@ def score(self, X, y):
         X, y = self._check_dimensions(X, y, predict=True)
         n_times, n_epochs, n_outputs = y.shape
         y_pred = self.predict(X)
-
-        y_pred = y_pred.reshape(y.shape, order='F')
-        y_pred = y_pred[self.keep_samples_]
-        y = y[self.keep_samples_]
+        y_pred = y_pred[self.valid_samples_]
```

Comment:
This is also within `score`, so I'm not sure how/why this would break `sklearn` compatibility.